### PR TITLE
VACMS-16217: Add initial income limits automation Workflow.

### PR DESCRIPTION
**Pull request by dsasser**
## Summary

Adds a Github Workflow for automating data sync between the Income Limits upstream data source (VES owned Oracle DB in the vagov-east VPC) to S3.

Why a self-hosted Github Action?

- This CANNOT be a Sidekiq script, because the Oracle Client Libraries are not compatible with Apple M1+ chipsets. See https://www.rubydoc.info/github/kubo/ruby-oci8/file/docs/install-on-osx.md.
- The Oracle database that contains the canonical dataset is private, and served from the vagov East VPC.
- Workflow allows local development without preventing Ruby bundles to be installed on Apple M1+ computers.

Existing CSVs (new files are not overwriting these)
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_county.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_gmtthresholds.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_incomethreshold.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_state.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_zipcode.csv

New CSVs (with temp filenames)
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_county_temp.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_gmtthresholds_temp.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_incomethreshold_temp.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_state_temp.csv
https://sitewide-public-websites-income-limits-data.s3.us-gov-west-1.amazonaws.com/std_zipcode_temp.csv

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16217

## Testing done
Testing in Github Workflow
Tested CSV files are uploaded to S3

## What areas of the site does it impact?
Income Limits
Github Workflows

## Acceptance criteria

- [x]  No error nor warning in the console.
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .github/workflows/income-limits-data-sync.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+permissions:
+  id-token: write
+  contents: read 
+jobs:
+  income_limits_data_sync:
+    runs-on: self-hosted
+    container:
+      image: public.ecr.aws/docker/library/ruby:3.2.2-bullseye
+      env:
+        SSL_CERT_FILE: /etc/ssl/certs/ca-certificates.crt
```
Comment (olivereri) [inline .github/workflows/income-limits-data-sync.yml:18]:
```suggestion
        SSL_CERT_FILE: /etc/ssl/certs/ca-certificates.crt
        NODE_EXTRA_CA_CERTS: /etc/ssl/certs/ca-certificates.crt
```

File: .github/workflows/income-limits-data-sync.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
+name: Income Limits Data Sync
+on:
+  push:
+  workflow_dispatch: null
+  schedule:
+    - cron: "35 0 1 * *"
+env:
+  BUCKET_NAME: "sitewide-public-websites-income-limits-data"
+  AWS_REGION: "us-gov-west-1"
```
Comment (olivereri) [inline .github/workflows/income-limits-data-sync.yml:9]: This is never used.

Comment (olivereri) [reply .github/workflows/income-limits-data-sync.yml:9]: Specifically AWS_REGION

**Review by olivereri — DISMISSED** [review]

**Review by Becapa — DISMISSED** [review]
This is my first time seeing a github workflow, but it looks good and makes sense. The ruby script looks good too and the comments are helpful.

**Review by Becapa — APPROVED** [review]

**Review by olivereri — APPROVED** [review]

**Pull request merged**